frontend/src/views/HomePage/components/UploadImageArea/UploadImage.styles.js

```diff
@@ -8,6 +8,7 @@ export const StyledUploadImage = styled.section`
   margin: 30px 50px;
   border-radius: 30px;
   background-color: ${({ theme }) => theme.colors.white};
+  border: 1px solid black;
```

Comment:
Why do you mix the usage of theme and non-theme colors? I don't think it's a good practice.

Reply:
Yes, that was an oversight.